src/main/java/me/liiot/snsserver/service/PostServiceImpl.java

```diff
@@ -54,6 +59,7 @@ public void uploadPost(User user, String content, List<MultipartFile> images) {
     }
 
     @Override
+    @Cacheable(cacheNames = CacheKeys.POST, key = "#postId")
```

Comment:
캐시 이름을 사용하는거라면 아래와 같은 형태가 되어야할 것 같은데 어떠신가요?

```suggestion
    @Cacheable(cacheNames = CacheNames.POST, key = "#postId")
```

Reply:
넵 수정하겠습니다.